modules/server/src/main/ts/bedrock/auto/Driver.ts

```diff
@@ -67,6 +69,16 @@ const addArguments = (capabilities: Record<string, any>, name: string, args: str
   capabilities[name].args = currentArgs.concat(args);
 };
 
+const getExtraBrowserCapabilities = (settings: DriverSettings): string[] => {
+  const extraCaps = settings.extraBrowserCapabilities.trim();
+  if (extraCaps.length > 0) {
+    return extraCaps.split(' ');
+  } else {
+    return [];
+  }
+};
+
```

Comment:
```suggestion
};
```
Just removing this line since it's causing linter errors:
```
  81:1  error  More than 1 blank line not allowed  no-multiple-empty-lines
```